contents/membership/_index/main.markdown

```diff
@@ -49,27 +67,30 @@ subscription.
 
 <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top" style="padding-left: 45px; padding-bottom: 15px;">
 <input type="hidden" name="cmd" value="_s-xclick">
-<input type="hidden" name="hosted_button_id" value="3PFMZ99CUZ7HU">
+<input type="hidden" name="hosted_button_id" value="6YY74YT8UKT9E">
 <table>
-<tr><td><input type="hidden" name="on0" value="HackerspaceSG monthly membership">HackerspaceSG <strong>monthly</strong> membership</td></tr><tr><td><select name="os0">
-	<!-- <option value="Sponsor">Sponsor : $2,048.00 SGD</option> -->
-	<!-- <option value="Patron">Patron : $1,024.00 SGD</option> -->
-	<option value="Resident">Resident : $512.00 SGD</option>
-	<option value="Hotdesk Priority">Hotdesk Priority : $256.00 SGD</option>
-	<option value="Hotdesk Regular">Hotdesk Regular : $128.00 SGD</option>
-	<option value="Youth/NSF">Youth/NSF : $64.00 SGD</option>
-	<option value="Advocate">Advocate : $32.00 SGD</option>
-	<option value="Backer">Backer : $16.00 SGD</option>
-	<!-- <option value="Friend">Friend : $8.00 SGD</option> -->
+<tr><td><input type="hidden" name="on0" value="HackerspaceSG monthly membership">HackerspaceSG monthly membership</td></tr><tr><td><select name="os0">
+	<option value="Regular">Regular : $128.00 SGD - monthly</option>
+	<option value="Associate">Associate : $64.00 SGD - monthly</option>
```

Comment:
I would like the most relevant tiers, imho 128 and 64 be at the top, and hence my chosen order